### PR TITLE
Fix: Add missing proguard rules

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
Fix: Add missing proguard rules

## :bulb: Motivation and Context
rules are per module, so some of them need to be placed in all of our android modules.
missing 2 important rules that keep our stack traces unambiguous

## :pencil: Checklist

- [X] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the docs if needed
- [X] No breaking changes

### CODE REVIEW

**Comment by bruno-garcia** [comment]
Seeing @maciejwalkowiak tagged here made me realize you probably want to tune the CODEOWNERS so the Android part is owned by manoel and the java part by maciej

File: sentry-android-timber/proguard-rules.pro [hunk 1]
```diff
@@ -2,4 +2,8 @@
 
 -keep class io.sentry.android.timber.** { *; }
 
+# To ensure that stack traces is unambiguous
+# https://developer.android.com/studio/build/shrink-code#decode-stack-trace
+-keepattributes LineNumberTable,SourceFile
```
Comment (bruno-garcia) [inline sentry-android-timber/proguard-rules.pro:7]: Doesn't this get through because of `android-core` already or do we need to specify it here also?

**Review by bruno-garcia — APPROVED** [review]
RSLGTM

File: sentry-android-timber/proguard-rules.pro (hunk 1 above)
Comment (marandaneto) [reply sentry-android-timber/proguard-rules.pro:7]: written in the PR template

> rules are per module, so some of them need to be placed in all of our android modules.

**Comment by marandaneto** [comment]
> Seeing @maciejwalkowiak tagged here made me realize you probably want to tune the CODEOWNERS so the Android part is owned by manoel and the java part by maciej

not sure, whos gonna review our own PRs then? :)

**Comment by bruno-garcia** [comment]
> > Seeing @maciejwalkowiak tagged here made me realize you probably want to tune the CODEOWNERS so the Android part is owned by manoel and the java part by maciej
> 
> 
> 
> not sure, whos gonna review our own PRs then? :)

You can assign ppl for review when needed. If you need me or Philipp to review tag us.
It's just too painful to get tagged on literally all PRs automatically

**Comment by bruno-garcia** [comment]
Also don't think it's good use of Maciejs time to get tagged in all android PRs. He doesn't care when we bump ndk.

**Comment by marandaneto** [comment]
> > > Seeing @maciejwalkowiak tagged here made me realize you probably want to tune the CODEOWNERS so the Android part is owned by manoel and the java part by maciej
> > 
> > 
> > not sure, whos gonna review our own PRs then? :)
> 
> You can assign ppl for review when needed. If you need me or Philipp to review tag us.
> It's just too painful to get tagged on literally all PRs automatically

are we talking about the mobile team or @maciejwalkowiak being tagged on an Android PR?

**Pull request merged**